modules/sdk-core/src/bitgo/inscription/iInscription.ts

```diff
@@ -0,0 +1,25 @@
+import { PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
+
+export type TxInfoUnspents = {
+  chain: number;
+  index: number;
+  witnessScript?: string;
+  id: string;
+  address: string;
+  value: number;
+  valueString: string;
+};
+
+export interface IInscription {
+  prepareReveal(inscriptionData: Buffer, contentType: string): Promise<PreparedInscriptionRevealData>;
+
+  // same response as wallet.submitTransaction
+  signAndSendReveal(
+    walletPassphrase: string,
+    tapLeafScript: TapLeafScript,
+    commitAddress: string,
+    unsignedCommitTx: string,
```

Comment:
`Buffer`, maybe even `UtxoPsbt`

Reply:
Curious, so that I understand this better, why do prefer to pass around a Buffer over a tx hex string?

Reply:
`string` is maybe the weakest type in JS. We want strongly typed not https://wiki.c2.com/?StringlyTyped